### PR TITLE
F#8 chair models

### LINKED ISSUE #8: Upload models of chair (007/008)
Upload blender models of chair into robert_gazebo/cad/ directory

Upload the visual models of chair in .dae extension into robert_gazebo/meshes/ directory

Upload the collision models of chair in .dae extension into robert_gazebo/meshes/ directory

Upload the collision models of chair in .sdf extension into robert_gazebo/models/ directory

Upload the collision models of chair in .config extension into robert_gazebo/models/classroom chair/ directory

### PR DESCRIPTION
**Pull request by LAnastasiia**
closes #8

### CODE REVIEW

File: robert_gazebo/worlds/ucu.world [hunk 1]
```diff
@@ -10,6 +10,18 @@
       <uri>model://sun</uri>
     </include>
 
+    <include>
```
Comment (AndriyPt) [inline robert_gazebo/worlds/ucu.world]: Please remove debug code

**Review by AndriyPt — CHANGES_REQUESTED** [review]
Please remove blend1 files

File: robert_gazebo/worlds/ucu.world (hunk 1 above)
Comment (LAnastasiia) [reply robert_gazebo/worlds/ucu.world]: Done

**Comment by AndriyPt** [comment]
Bad scale of the objects
image

**Comment by LAnastasiia** [comment]
Done

**Review by AndriyPt — APPROVED** [review]

**Pull request merged**